enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/TemporalAcceptanceTest.scala

```diff
@@ -450,13 +452,13 @@ class TemporalAcceptanceTest extends ExecutionEngineFunSuite with QueryStatistic
 
   test("should not provide undefined accessors for date") {
     shouldNotHaveAccessor("date", Seq("hour", "minute", "second", "millisecond", "microsecond", "nanosecond",
-      "timezone", "offset", "epoch",
+      "timezone", "offset", "offsetMinutes", "epochSeconds", "epochMillis",
```

Comment:
You can add `epochSecond` and `epochMillis` to the `time` test also. I think that was overlooked before.